frontend/src/types/index.d.ts

```diff
@@ -349,3 +349,10 @@ export interface IWhat {
   single: string;
   plural: string;
 }
+
+export interface IGDXList {
+  // todo: Define a good type. "Any" type temporarily permitted.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  blocks: Array<[{ label: string; value: any }]>;
```

Comment:
just curious what values you would be expecting here, is it possible to narrow it down from any to ie string | number, or what ever it needs to be.